keyboards/keycapsss/kimiko/rev1/info.json

```diff
@@ -15,11 +15,18 @@
     "diode_direction": "COL2ROW",
     "encoder": {
         "rotary": [
-            {"pin_a": "F4", "pin_b": "F5"}
+            {"pin_a": "F5", "pin_b": "F4"}
         ]
     },
     "split": {
-        "soft_serial_pin": "D2"
+        "soft_serial_pin": "D2",
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "F4", "pin_b": "F5"}
```

Comment:
Can you change it to

```json
{"pin_a": "F5", "pin_b": "F4"}
```

Reply:
In that case, are you suggesting the the _other_ hand be:

```json
{"pin_a": "F4", "pin_b": "F5"}
```

?

If they're _both_

```json
{"pin_a": "F5", "pin_b": "F4"}
```

then we end up with the encoder direction being reversed for one of the hands.